pkg/testing/integration/program.go

```diff
@@ -702,13 +713,30 @@ func (pt *programTester) runPulumiCommand(name string, args []string, wd string)
 		}
 	}
 
-	runErr := pt.runCommand(name, cmd, wd)
+	_, _, err = retry.Until(context.Background(), retry.Acceptor{
+		Accept: func(try int, nextRetryTime time.Duration) (bool, interface{}, error) {
```

Comment:
Should we log something - or somehow else indicate there was a failure and retry?

Reply:
Sure, that's reasonable. I think the stderr output will still be present in the logs, but I'm not sure.